CHANGELOG.md

```diff
@@ -65,6 +65,8 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
   `wharfapi.Provider.UploadURL`, which will be removed in wharf-api v5.0.0 as it
   did not provide any functionality. (#35)
 
+- Changed docker-build scripts for easier windows building. (#44)
```

Comment:
nit: Could use the filename "Dockerfile" instead of "docker-build scripts" to be more precise. IMO I understand "Dockerfile" better

```suggestion
- Changed Dockerfile for easier windows building. (#44)
```